packages/editor/src/lib/editor/Editor.ts

```diff
@@ -7389,9 +7390,9 @@ export class Editor extends EventEmitter<TLEventMap> {
 	 *
 	 * @public
 	 */
-	setStyleForSelectedShapes<T>(
-		style: StyleProp<T>,
-		value: T,
+	setStyleForSelectedShapes<S extends StyleProp<any>>(
+		style: S,
+		value: StylePropValue<S>,
```

Comment:
why were these changes needed here? seems like we've added a lot of `any`s

Reply:
We discussed this offline. It's some kind of variance thing. I couldn't figure it out but also I don't think it matters since the `any`s don't leak into values, and I don't think the generic type gives us much (if any?) extra safety for these style props.